openadapt/crud.py

```diff
@@ -22,10 +29,15 @@
 memory_stats = []
 
 
+def _insert(event_data: dict, table: sa.Table, buffer: list = None) -> Any | None:
```

Comment:
Should `buffer` be:

```
buffer: list[Any] | None = None
```

?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507